app/utils/estimate_prioritization/estimate_prioritization.py

```diff
@@ -0,0 +1,113 @@
+import pandas as pd
+from statsmodels.stats.proportion import proportion_confint
+from app.utils.estimate_prioritization import prioritization_criteria_testunadj, \
+    prioritization_criteria_testadj, pooling_function_maps
+
+
+def get_pooled_estimate(estimates):
+
+    if estimates.shape[0] == 1:
+        pooled = estimates.iloc[0]
+    else:
+        # use the estimate with the max denominator as the 'base' for the pooled estimate
+        pooled = estimates.loc[estimates['denominator_value'].idxmax(axis = 1)] 
+
+        # pool all values that can be pooled
+        for (summary_type, cols_to_summarize, summary_function) in pooling_function_maps:
+            if summary_function is not None:
+                valid_cols_to_summarize = set(cols_to_summarize) & set(estimates.columns)
+
+                for col in valid_cols_to_summarize:
+                    pooled.at[col] = summary_function(estimates, col)
```

Comment:
why .at and not .loc?

Reply:
pooled is a series and not a DF. @rahularoradfs can you confirm

Reply:
confirmed